pkgs/misc/tmux-plugins/default.nix

```diff
@@ -64,6 +64,17 @@ in rec {
     };
   };
 
+  better-mouse-mode = mkDerivation {
+    pluginName = "scroll_copy_mode";
+    version = "unstable-2017-10-16";
+    src = fetchFromGitHub {
+      owner = "NHDaly";
+      repo = "tmux-better-mouse-mode";
+      rev = "aa59077c635ab21b251bd8cb4dc24c415e64a58e";
+      hash = "sha256:nPNa3JvDgptGvy2vpo0WSZytyu7kFSEn/Jp/OGA0ZBg=";
+    };
+  };
```

Comment:
It appears that this plugin is unmaintained and may not work with tmux 3.0 (which we now have):

https://github.com/NHDaly/tmux-better-mouse-mode/issues/41

> huh, good to know. Maybe this package can just be parked if it's not really needed on 3.0 anymore? That would be great! This only existed as a stopgap while waiting for better mouse support in tmux itself anyway

If you need this, it might be more suited for the NUR: https://github.com/nix-community/NUR

Reply:
I wasn’t aware that the plugin was unmaintained, I’ll close this PR.